src/main/scala/com/databricks/spark/avro/AvroOutputWriter.scala

```diff
@@ -104,7 +104,11 @@ private[avro] class AvroOutputWriter(
       case DateType => (item: Any) =>
         if (item == null) null else item.asInstanceOf[Date].getTime
       case ArrayType(elementType, _) =>
-        val elementConverter = createConverterToAvro(elementType, structName, recordNamespace)
+        val elementConverter = createConverterToAvro(
+          elementType,
+          structName,
+          SchemaConverters.getNewRecordNamespace(elementType, recordNamespace, structName)
+          )
```

Comment:
put this `)` to the above line
https://docs.scala-lang.org/style/indentation.html#methods-with-numerous-arguments
Thanks!